kbase-extension/static/kbase/js/widgets/narrative_core/kbaseNarrativeDataList.js

```diff
@@ -1492,9 +1492,11 @@ define([
                 .append('type')
                 .on('click', function () {
                     self.sortData(function (a, b) {
-                        if (self.dataObjects[a].info[2].toUpperCase() > self.dataObjects[b].info[2].toUpperCase())
+                        let aType = self.dataObjects[a.objId].info[2].toUpperCase().match(/\.(.+)/)[1];
```

Comment:
Sadly, we're still in ES5 (and there's no Babel installed on the build path yet...), so "let" is what's causing the build to fail.

Reply:
That took me entirely too long to track down. Sorry!